src/Stripe.net/Services/PaymentMethods/PaymentMethodCreateOptions.cs

```diff
@@ -0,0 +1,62 @@
+namespace Stripe
+{
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class PaymentMethodCreateOptions : BaseOptions
+    {
+        /// <summary>
+        /// Billing information associated with the PaymentMethod that may be used or required by
+        /// particular types of payment methods.
+        /// </summary>
+        [JsonProperty("billing_details")]
+        public BillingDetailsOptions BillingDetails { get; set; }
+
+        /// <summary>
+        /// If this is a <c>card</c> PaymentMethod, for backwards compatibility, you can  provide a
+        /// Stripe token (e.g., for apple pay, amex checkout, or legacy checkout).
+        /// </summary>
+        [JsonProperty("card")]
+        public string CardTokenId { get; set; }
```

Comment:
This API takes a token id for legacy reasons, or a hash (for PCI compliant users + .net mobile apps). Are we still okay taking that approach?
The alternative could be to not support this in .NET so not sure what's best.

I also put my own comments there since the ones in the docs would not work.

Reply:
Ah I see. Hmm so there's a valid use case of passing raw PANS specifically at this new endpoint (mobile apps)? On previous decision we assume that merchants passing raw PANS will already be doing custom integration anyways right? But on the .net mobile apps, can we make exception only for this new PaymentMethod, i.e can we enforce to passing token in old endpoints with params  `source` and `external_account`, and not this?

Reply:
I think it's fine to keep the existing code for now. We will do a major version in the future that removes this ability on endpoints that should be called server-side. But PaymentMethod creation can happen client-side in a mobile application and since we don't have a .NET mobile SDK, users rely on stripe-dotnet for this.